Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs/Program.cs

```diff
@@ -860,26 +871,9 @@ select new
                 inherited: false,
                 isDefault: true);
 
-            // Step 2: Process from bottom to top (least to most in terms of priority)
-
-            // Are there any explicit settings on the resource for the current actiona and claim set?
-            ActionAndStrategy explicitActionAndStrategyForAction;
-
-            if (localActionAndStrategy.ExplicitActionAndStrategyByClaimSetName.TryGetValue(
-                claimSetName,
-                out explicitActionAndStrategyForAction))
-            {
-                // There was a claim set specific setting on the current resource
-                effectiveActionAndStrategy.TrySetActionGranted(inherited: false);
-
-                effectiveActionAndStrategy.TrySetAuthorizationStrategy(
-                    explicitActionAndStrategyForAction.AuthorizationStrategy,
-                    resource.Name,
-                    inherited: false,
-                    isDefault: false);
-            }
+            // Step 2: Process from top to bottom (least to most in terms of priority)
```

Comment:
Had to invert the calculation of the overrides here since it was prioritizing the inherited override instead of the override that is directly associated to the resource.